neofs-testlib/neofs_testlib/env/env.py

```diff
@@ -419,7 +451,7 @@ def __getstate__(self):
         del attributes["process"]
         return attributes
 
-    def start(self):
+    def generate_network_config(self):
```

Comment:
`generate_common_network_config`? I really expected IR config to be generated here, but...

Reply:
This is the config that we use for neofs-adm commands. It is not really common since each instance of InnerRing has its own network config, mostly with same variables, but still separate files.